.github/workflows/provider-publish-service-artifacts.yml

```diff
@@ -0,0 +1,107 @@
+name: Provider Publish Service Artifacts
+
+on:
+  workflow_call:
+    inputs:
```

Comment:
Could you please also add the following input parameters to this workflow?
- `xpkg_reg_orgs`: Required, default value: `xpkg.upbound.io/upbound`. We should pass the value of this parameter in `Publish Artifacts` & `Build Artifacts` steps to the `make` command: `make -j $num_packages SUBPACKAGES="${{ inputs.subpackages }}" XPKG_REG_ORGS=${{ inputs.xpkg_reg_orgs }} XPKG_REG_ORGS_NO_PROMOTE =${{ inputs.xpkg_reg_orgs }} build.all`.
- `version`: Required, no default (ideally we would like to make `version` optional and only set the `VERSION` make variable if the `version` Github workflow input parameter is specified but we can do this in a follow-up PR). Again we need to set the `VERSION` make variable in both `Publish Artifacts` & `Build Artifacts` steps.